packages/fxa-content-server/app/scripts/templates/settings.mustache

```diff
@@ -5,7 +5,10 @@
     </div>
   {{/ccExpired}}
   <header id="fxa-settings-header">
-    <h1 id="fxa-manage-account"><span class="fxa-account-title">{{#t}}Firefox Accounts{{/t}}</span></h1>
+    <div class="header-lockup">
+      <h1 id="fxa-manage-account"><span class="fxa-account-title">{{#t}}Firefox Accounts{{/t}}</span></h1>
+      <a href="/beta/settings" class="nu-settings-button">Switch to new design</a>
```

Comment:
The button down state shows the text as underlined. I don't think we want that for a "button"